### PR TITLE
Add dbt-fal adapter

### PR DESCRIPTION
**Pull request by chamini2**
## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/reference/warehouse-setups/fal-setup.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: "fal setup"
+meta:
+  maintained_by: fal.ai
+  authors: 'Features & Labels (https://github.com/fal-ai)'
+  github_repo: 'fal-ai/fal'
+  pypi_package: 'dbt-duckdb'
```
Comment (burkaygur) [inline website/docs/reference/warehouse-setups/fal-setup.md]: is this correcT?

**Review by dataders — CHANGES_REQUESTED** [review]
@chamini2 you'll also want to add yourselves to the table on `docs/supported-data-platforms` 
https://docs.getdbt.com/docs/supported-data-platforms#community-adapters

**Review by dataders — DISMISSED** [review]
🐿️

**Review by dataders — APPROVED** [review]
thanks @chamini2! :shipit:

**Pull request merged**